Get photos from buyed album

gabrielbori opened the pull request:
## Summary

Se desea que un usuario pueda listar las fotos de un album que previamente haya comprado al proveedor de albumes

Criterio de aceptación:
Para listar las fotos, un usuario debe poder realizar un http request (GET) a "users/albums/:id/photos"
Se debe estar autenticado para consumir dicho recurso
Un usuario solo podra ver las fotos de albumes que el haya comprado
Un administrador solo podra ver las fotos de albumes que haya comprado
Se deben agregar Tests, para los cuales se debera mockear la respuesta del servicio externo
Realizar logs informativos en caso de considerarlo necesario.
Loggear un error en caso de falla de la base de datos.
Se debe deployar la nueva funcionalidad a Heroku y verificar que funcione correctamente.

## Known Issues

[Change!] List any known issue of the feature you are implementing.

## Trello Card

[Change!] Link to the associated Trello card.

gabrielbori committed:
Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

gabrielbori committed:
Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

gabrielbori committed:
Finished tests

gabrielbori committed:
Admin user support.

gabrielbori committed:
Finished tests

gabrielbori committed:
Admin user support.

gabrielbori committed:
Finished tests

gabrielbori committed:
Finished tests

gabrielbori committed:
Admin user support.

gabrielbori committed:
Admin user support.

gabrielbori committed:
Tests

gabrielbori committed:
fix merge problems

gabrielbori committed:
fix merge problems

gabrielbori committed:
fix merge and rebase problems

gabrielbori committed:
Update errorsMessages.js

gabrielbori committed:
Update errorsMessages.js

gabrielbori committed:
Fixed again some merge and rebase issue, hope it's last time·

gabrielbori committed:
Fixed again some merge and rebase issue, hope it's last time·

gabrielbori committed:
Finished buy album logic

gabrielbori committed:
Finished buy album logic

gabrielbori committed:
Changed auth way to authenticate

gabrielbori committed:
Changed auth way to authenticate

gabrielbori committed:
Added roles to admin user, modified tests

gabrielbori committed:
Added roles to admin user, modified tests

gabrielbori committed:
pr fixes

gabrielbori committed:
pr fixes

gabrielbori committed:
deleted comment

gabrielbori committed:
deleted comment

gabrielbori committed:
Delete password from user attatched to request

gabrielbori committed:
Delete password from user attatched to request

gabrielbori committed:
Started with tests, minor fixes, rebased

gabrielbori committed:
Almost finished tests, some errors ocurring while runing them

gabrielbori committed:
fixed Tests

gabrielbori committed:
merged with master

gabrielbori committed:
Changed process env url search

gabrielbori committed:
finished some fixes

gabrielbori committed:
added done

gabrielbori committed:
Fixed pr issues

gabrielbori committed:
Changed order to buy album. Refactor album by user model, took out id

gabrielbori committed:
Created logic to get albums buyed by user id

gabrielbori committed:
Try other thing

gabrielbori committed:
Added dones to see if travis stops throwing errors

gabrielbori committed:
Try other thing

gabrielbori committed:
took out done

gabrielbori committed:
finished tests

gabrielbori committed:
added catch to see error on tests

gabrielbori committed:
Changed emal createion to avoid chance error

gabrielbori committed:
Added endpoint to get photos from buyed album

gabrielbori committed:
working on tests

gabrielbori committed:
Added empty lines, changed serializer response of albums buyed by user

gabrielbori committed:
Buy albums (#15)

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* Finished buy album logic

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Finished buy album logic

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Almost finished tests, some errors ocurring while runing them

* fixed Tests

* Changed process env url search

* finished some fixes

* added done

* Fixed pr issues

* Changed order to buy album. Refactor album by user model, took out id

Added empty lines, changed serializer response of albums buyed by user

Pulled from master

gabrielbori committed:
Merge branch 'list-buyed-albums-by-userId' of https://github.com/gaboxxz/gb-express-js into list-buyed-albums-by-userId

gabrielbori committed:
merge

emanuelcasco commented on app/controllers/albums.js:
```diff
@@ -47,3 +49,33 @@ exports.buyAlbum = (req, res, next) => {
+    .getPhotosFromAlbumByIdAndUser(albumId, userId)
+    .then(photosList => {
+      res.send(serializeAlbumPhotos(photosList));
+    })
```
Dont forget return statement

emanuelcasco commented on app/serializers/albums.js:
```diff
@@ -0,0 +1,16 @@
+  };
+  return serializedPhotoList;
+};
+exports.serializeAlbumsResponse = findedAlbums => {
```
Avoid side effects, this function should receive and object, and create a new one without modify original

gabrielbori committed:
Modified tests with required changes in pr

gabrielbori committed:
List buyed albums by user (#16)

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* Finished buy album logic

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Finished buy album logic

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Almost finished tests, some errors ocurring while runing them

* fixed Tests

* Changed process env url search

* finished some fixes

* added done

* Fixed pr issues

* Changed order to buy album. Refactor album by user model, took out id

* Created logic to get albums buyed by user id

* Added dones to see if travis stops throwing errors

* Try other thing

* took out done

* finished tests

* added catch to see error on tests

* Changed emal createion to avoid chance error

* Added empty lines, changed serializer response of albums buyed by user

* Buy albums (#15)

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* Finished buy album logic

* Admin user support.

* Finished tests

* Admin user support.

* Finished tests

* Users list (#12)

* Added users middleware, post for users signUp

* add endpoint and validations to create users

* deleted empty file

* Add log when field valitation fails

* fixed some checks in user middleware, add error handling

* Fix some names

* Fix error, will change un other branch

* tests

* Update user.js

* Add migrations tests

* Finished signUp tests

* Added test migrations, deleted extra file

* Working on signIn feature, added token response

* Add enter, delete extra primise.resolve

* Update signUp.js

* Update users.js

* Fix serveral tests. Fix error messages

* Deleted coments

* Added SECRET to travis file

* Update .travis.yml

* Added middleware logic. Added token verification. Still need some changes

* Added variable validUser to reuse in tests

* Changed test to use validuser fields

* Fix errors in user controller

* Changed error handling

* changes in validations and errors

* Delete console.log

* changes in constants and error messages. Added helpers folder

* Add created user serializer

* Now using funcion to creaate user

* Changed logic to test email existance, add mapper to hash password

* Changed logic to test email existance, add mapper to hash password

* fix error in mapper

* fix error in mapper

* Changed constants to use snake_case

* re-doing tests with new changes from signUp

* Changed messages from param validator

* changed email already exists message

* Update users.js

* modifing tests

* Modify tests to match new error formatt

* Fixed last test to match new error handling

* change logic to work with schemas and helpers

* fix tests

* todo

* Moved eslint disable, deleted paramErrors empty object2

* Added try catch to db calls

* Fix error in returning userData when created

* Changed constants to env file. created serializer for param errors

* Changed constants to env file. created serializer for param errors

* solved require issue

* solved problem with serialize function call

* Changed env variables to use config idex and testing

* Fixes in email check error

* changed email already exists test

* changed name of findBy email function

* Pretier functions. Added serializers and enhance errors

* Fixed tests

* Changed email registered error handle

* Chaanged test to match new email error handler

* Changed error function when email does not exist. Added log

* added first test to get users list

* deleted commented function

* Added factory-girl use

* Added more tests

* Added some tests and fixed other merge issues

* Finished endpoint to get users list being authorized

* change log call

* Changed user service to services folder because could not run npm console

* Users list pr modifications

* Users list pr modifications

* Renamed variables

* fix merge problems

* Update errorsMessages.js

* Fixed again some merge and rebase issue, hope it's last time·

* Finished buy album logic

* Changed auth way to authenticate

* Added roles to admin user, modified tests

* pr fixes

* deleted comment

* Delete password from user attatched to request

* Almost finished tests, some errors ocurring while runing them

* fixed Tests

* Changed process env url search

* finished some fixes

* added done

* Fixed pr issues

* Changed order to buy album. Refactor album by user model, took out id

Added empty lines, changed serializer response of albums buyed by user

Pulled from master

* Modified tests with required changes in pr

Add return statement

Merged master

gabrielbori committed:
Merge branch 'master' into get-photos-from-buyed-album

gabrielbori committed:
fix deleted path

arinaldi118 commented on app/interactors/albums.js:
```diff
@@ -22,3 +22,17 @@ exports.userBuysAlbum = (albumId, userId) =>
+    })
+    .then(albumByUser => {
+      if (!albumByUser) {
+        // TODO: take message to constant
```
:see_no_evil:

arinaldi118 commented on app/serializers/albums.js:
```diff
@@ -1,10 +1,21 @@
+  return serializedAlbums;
+};
+
+exports.serializeAlbumPhotos = photosList => {
```
`exports.serializeAlbumPhotos = photosList => ({ ... });`

arinaldi118 commented on test/getPhotosFromBuyedAlbumByUser.js:
```diff
@@ -0,0 +1,177 @@
+const { factory } = require('factory-girl');
+const errors = require('../app/errors');
+const { roles } = require('../app/constants/roles');
+// const constants = require('../app/constants');
```
delete comments

arinaldi118 commented on test/getPhotosFromBuyedAlbumByUser.js:
```diff
@@ -0,0 +1,177 @@
+          .post('/users/sessions')
+          .send({ email: validUser.email, password: validUser.password })
+          .then(res => {
+            // eslint-disable-next-line prefer-destructuring
```
Delete this

arinaldi118 commented on test/getPhotosFromBuyedAlbumByUser.js:
```diff
@@ -0,0 +1,177 @@
+          .post('/users/sessions')
+          .send({ email: validUserAdmin.email, password: validUserAdmin.password })
+          .then(res => {
+            // eslint-disable-next-line prefer-destructuring
```
Idem

arinaldi118 commented on test/getPhotosFromBuyedAlbumByUser.js:
```diff
@@ -0,0 +1,177 @@
+          .query(true)
+          .reply(200, mockPhotosAlbunId);
+      })
+      .catch(err => {
```
Delete this catch

gabrielbori committed:
Deleted coments, Changed serializer, Deleted catch in test
